### ISSUE TITLE
What is the loss function you used in the 'vot_SiamRPN'

### ISSUE THREAD

**Issue by ZhangSanFengByGit** [foolwood/DaSiamRPN#29]
@foolwood hello~
What is the loss function you used in the 'vot_SiamRPN', when you count the distance between res[x,y,w,h] and the label_position.   Is that cross entrophy loss or?

**Comment by foolwood** [comment]
@ZhangSanFengByGit 
The loss functions are consistent with sec3.2 of the origin [paper](http://openaccess.thecvf.com/content_cvpr_2018/papers/Li_High_Performance_Visual_CVPR_2018_paper.pdf).

**L\_cls is the cross-entropy loss and L\_reg is smooth L\_1.**
Thanks for your question.
